chroniques-enyllion/src/assets/main.css

```
@@ -0,0 +1,183 @@
+/* Polices médiévales-fantasy de l'identité Enyllion */
+@import url('https://fonts.googleapis.com/css2?family=Cinzel+Decorative:wght@400;700;900&family=Eagle+Lake&family=Macondo&display=swap');
```

Stylelint (17.14.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Cinzel+Decorative:wght@400;700;900&family=Eagle+Lake&family=Macondo&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Cinzel+Decorative:wght@400;700;900&family=Eagle+Lake&family=Macondo&display=swap'" (import-notation)

(import-notation)